### DISCUSSION TITLE
PDA only confusion.

### DISCUSSION THREAD

**Discussion by Bigfellas** [aqualinkd/AqualinkD#265] [Q&A]
When I started to research this I thought I was ok because I had a panel with  RS wiring digrame stuck to it. So thought I was fine.
Now that I have the hardware I got  the serial logger to run and got this report. So I assume this is telling me this is a PDA only unit and from what I am reading probably not worth continuing especially as I have a  VSP  as well.  Can anyone confirm this for me please .

```
AqualinkD serial_logger V2.0
Notice:  RS Serial: Port /dev/ttyUSB0 low latency mode is set
Notice:  RS Serial: Logging serial information!
Please wait.|

Notice:  RS Serial: RS485 interface received 600 packets in 66 seconds (~9.09 Msg/Sec)
Notice:  RS Serial: Jandy Control Panel Model   :  PDA-P8 Only
Notice:  RS Serial: Jandy Control Panel Version :  PPD PDA. 5.0
Notice:  RS Serial: Jandy ID's found
Notice:  RS Serial: ID 0x63 is not used  <-- can use for Aqualinkd (PDA mode only)
Notice:  RS Serial: ID 0x60 is not used  <-- can use for Aqualinkd (PDA mode only)
Notice:  RS Serial: ID 0x61 is not used  <-- can use for Aqualinkd (PDA mode only)
Notice:  RS Serial: ID 0x62 is not used  <-- can use for Aqualinkd (PDA mode only)
Notice:  RS Serial: ID 0x78 is in use    <-- Jandy VSP ePump
Notice:  RS Serial:

Notice:  RS Serial: Suggested aqualinkd.conf values
Notice:  RS Serial: -------------------------
Notice:  RS Serial: panel_type =  PDA-P8 Only
Notice:  RS Serial: device_id = 0x63
Notice:  RS Serial: -------------------------
pi@aqualinkd:~/software/AqualinkD $
```

**Comment by sfeakes** [comment]
Wiring is the same between both types of panel.
You do have PDA panel.
You may as well continue since you have got this far. Many people use this for PDA only panels and it works for them, others don’t like the limitations and convert to a full RS panel.

**Reply by Bigfellas** [reply]
Thanks for the confirmation. Looks like I have the web interface configured and working. I will see how it goes with home assistant. Thanks for the  hard work you have put into the devlopment @sfeakes

**Comment by gottalovetrees** [comment]
Hello, I am in a very similar situation, without a VSP. If I choose to upgrade my panel, what do I need to actually purchase? Just the board with the correct chip?  like this https://www.ebay.com/itm/187556114989?  

EDIT: Upon further inspection, I don't see where that would mount on my current board. I'm guessing the entire board would have to be replaced.. like thishttps://www.ebay.com/itm/267440406478?  Is that correct?

I am so thankful that this project exists. Thank you for all that you do.  I help with a family vacation home and the PDA (for a PDA only system) has been swimming and destroyed multiple times.  This is the perfect solution, thank you.  I wish I was a better programmer than I am but I am learning.

**Reply by AUPoolUser** [reply]
What version of PDA do you have? You may just need to change the chip? if this is the case you can upgrade to a maximum version of MMM. Not sure if MMM version of AqualinkD RS will work with VSPs - I’ve been on the outer of this project for some time.

**Reply by gottalovetrees** [reply]
Thank you for the reply.

I'll include these 2 pictures because I'm not sure which you are asking for. 
<img width="449" height="587" alt="image" src="https://github.com/user-attachments/assets/50fc9349-af61-4721-85a6-2a9e3e91be66" />

<img width="1248" height="831" alt="image" src="https://github.com/user-attachments/assets/125e1e91-0881-47d9-b0be-50ae3d77096b" />

**Comment by AUPoolUser** [comment]
You have a Generation 2: 52-PIN PPD Chip Socketted Board (PCB#8194) - this was used Approximately between 2002–2006. You can change your PDA system to an RS system to a maximum version of MMM by buying something like this below. I have bought from this supplier. He is very knowledgeable. I would contact him. 

BUT I am not sure VSPs work with firmware MMM circa 2006???

https://www.ebay.com/itm/135617124876?_skw=mmm&epid=2155739761&itmmeta=01K8RSX0M8CZ9GP4SK74PBVNCB&hash=item1f93691a0c:g:tNMAAeSwWTFn0Uhb

You may be better off buying a bezel upgrade - do a search for Jandy IQ20RS upgrade kit (circa 2017) or Jandy IQ30RS upgrade kit (circa 2023). You get iAqualink, the antenna and a new Bezel - the bezel is basically a new computer for your Aqualink and makes it modern. 

Having said that. I am still using MMM - but I don’t need VSP control. 

Hope this helps.

**Comment by sfeakes** [comment]
Rev MMM doesn’t support VSP.
The chip upgrade is the cheapest option, but If you want VSP your best bet is to buy a T.2 or newer panel.
you can replace the whole PDA board with an RS board like this.  https://www.ebay.com/itm/336253887218?

**Reply by jkosharek** [reply]
What’s the latest Panel version? I have a RS 16 P & S Rev T and it doesn’t have the virtual outputs.

Mine also crashes when I select the light color for the Spa/Pool the lights on the swim shelf and accent change fine. I was thinking about switching the color changing to Aqualinkd but haven’t gotten around to it.

**Reply by AUPoolUser** [reply]
I believe it’s Revision Y which you get if you buy the latest iQ30 Upgrade Kit.
